pom.xml

```diff
@@ -58,6 +58,15 @@
             <artifactId>mapstruct</artifactId>
             <version>${mapstruct.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-core</artifactId>
+            <version>${liquibase.version}</version>
```

Comment:
you don't need to specify liquibase version - it comes from `spring-boot-starter-parent`